moveit_core/CMakeLists.txt

```diff
@@ -202,6 +202,6 @@ add_subdirectory(planning_interface)
 add_subdirectory(planning_request_adapter)
 add_subdirectory(trajectory_processing)
 add_subdirectory(distance_field)
-add_subdirectory(collision_distance_field)
+#add_subdirectory(collision_distance_field)
```

Comment:
Because `collision_distance_field_robot` and `collision_distance_field_world` also need to be merged in the same way as the other files in this PR, but it's not done yet. @j-petit mentioned it in the call although it's not written out here.